core/quota/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package quota_test
```

Comment:
I think we should have a general rule that _package_test.go_ is local to the package.